src/controllers/multer.js

```diff
@@ -0,0 +1,32 @@
+const multer = require('multer')
+const path = require('path');
+
+const DIR = './public/'
+
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, DIR)
+  },
+  filename: function (req, file, cb) {
+    const uniqueSuffix = Date.now() + '-' + Math.round(Math.random() * 1E9 ) + path.extname(file.originalname)
+    cb(null, file.fieldname + '-' + uniqueSuffix)
+  }
+})
+
+
+const upload = multer({
```

Comment:
I moved the callback function to multer.js from the server. Since this "upload" function is not called from the server anymore, I kept the variable name as it is. Let me know if you think we should change it!